packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -415,20 +415,25 @@ class DeepLinksController extends DisposableController {
   }
 
   Future<void> validateLinks() async {
-    List<LinkData> linkdata = _allRawLinkDatas;
-    if (linkdata.isEmpty) {
+    final appLinkSettings = currentAppLinkSettings;
+    if (appLinkSettings == null) {
```

Comment:
if appLinkSettings!=null but linkdata is empty, should also be PagePhase.noLinks

Reply:
added a isEmpty check in line 429